### PR TITLE
set TLSMinVersion for webhook servers

### PR DESCRIPTION
**Pull request by mythi**
Fixes: #1039
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

**Review by bart0sh — COMMENTED** [review]
/lgtm

File: test/e2e/utils/utils.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Intel Corporation. All Rights Reserved.
+// Copyright 2020-2022 Intel Corporation. All Rights Reserved.
```
Comment (hj-johannes-lee) [inline test/e2e/utils/utils.go:1]: What about the copyright years of the files below?
Do we change?
- cmd/fpga_admissionwebhook/main.go
- cmd/sgx_admissionwebhook/main.go
- test/e2e/sgxadmissionwebhook/sgxaadmissionwebhook.go

Comment (mythi) [reply test/e2e/utils/utils.go:1]: fixed

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**